package.json

```diff
@@ -18,12 +18,15 @@
     "@orca-so/common-sdk": "^0.1.12",
     "@orca-so/whirlpools-sdk": "^0.8.2",
     "@project-serum/anchor": "^0.26.0",
+    "@raydium-io/raydium-sdk": "^1.3.1-beta.8",
     "@solana/web3.js": "^1.70.1",
     "@types/bn.js": "^5.1.1",
     "@types/node": "^18.11.18",
+    "bs58": "^5.0.0",
     "cors": "^2.8.5",
     "express": "^4.18.2",
     "express-prom-bundle": "^6.6.0",
+    "js-sha256": "^0.9.0",
```

Comment:
as this is running on node, probably better to use a polyfill that resolves to node-crypto like "sha256"

Reply:
anchor uses crypto-hash internally

Reply:
looks like anchor uses @noble/hashes/sha256